uaclient/config.py

```diff
@@ -544,7 +544,9 @@ def _attached_status(self) -> "Dict[str, Any]":
                 "origin": contractInfo.get("origin"),
                 "subscription": contractInfo["name"],
                 "subscription-id": contractInfo["id"],
+                "contract-created-at": contractInfo["createdAt"],
                 "notices": self.read_cache("notices") or [],
+                "products": contractInfo["products"],
             }
```

Comment:
I was expecting the structure change to happen up in this function I think.

Currently this function returns an internal representation of the data that `format_json_status` interprets and restructures - which feels like an extra step.

What do you think of returning the new structure from this function? @lucasmoura

Reply:
It makes sense. I was trying to avoid changing the internal representation of the data, but there is no good reason to keep it as is. I will update the code